### PR TITLE
add BCM 2024 workshop

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

**Review by oruebel — APPROVED** [review]

File: Workshop_2024_BCM/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  * Extensions
+  * Analysis and visualization tools
+  * Other NWB Events
+* 12-1: Networking lunch
+* 1-2: Converting data to NWB
+  * NWB GUIDE
+  * NeuroConv
+  * PyNWB and MatNWB APIs
+  * Extensions
+* 2-3: Publishing data the DANDI Archive
```
Comment (rly) [inline Workshop_2024_BCM/README.md:26]:
```suggestion
* 2-3: Publishing data on the DANDI Archive
```

File: Workshop_2024_BCM/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  * Extensions
+  * Analysis and visualization tools
+  * Other NWB Events
+* 12-1: Networking lunch
+* 1-2: Converting data to NWB
+  * NWB GUIDE
+  * NeuroConv
+  * PyNWB and MatNWB APIs
+  * Extensions
+* 2-3: Publishing data on the DANDI Archive
```
Comment (rly) [inline Workshop_2024_BCM/README.md:26]:
```suggestion
* 2-3: Publishing and accessing data on the DANDI Archive
```

File: Workshop_2024_BCM/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  * Extensions
+* 2-3: Publishing and accessing data on the DANDI Archive
+  * Uploading data
+  * DANDI Metadata
+  * Neurosift
+  * DANDI Hub
+  * Dendro
+  * DANDI API
+
+**April 16, 9-3: NWB and DANDI hacking & office hours.** Participant will bring their own data and work with the NWB and DANDI teams to convert their data to NWB and publish it on DANDI.
```
Comment (rly) [inline Workshop_2024_BCM/README.md:34]:
```suggestion
**April 16, 9am - 3pm: NWB and DANDI hacking & office hours.** Participant will bring their own data and work with the NWB and DANDI teams to convert their data to NWB and publish it on DANDI.
```

**Comment by rly** [comment]
Looks great to me! The dates to sign up for in the form were wrong so I fixed those also.

**Pull request merged**